src/components/OnBoardingLayout/Admin/DesktopLayout.js

```diff
@@ -259,7 +260,7 @@ const MainContentDesktop = props => (
         >
           <StyledButton
             forward
-            href="https://geralnafilapt.typeform.com/to/VtDUdM"
+            onClick={() => (window.location.href = ADMIN_SIGNUP_PATH)}
```

Comment:
This isn't semantically correct, the Button shouldn't be used as an anchor. If the `href` property is valid in this component use that instead or Style an anchor tag accordingly to have the required styles.